docs/install.md

```diff
@@ -12,32 +12,30 @@ pip install pymmcore-plus
 pip install "pymmcore-plus[cli]"
 ```
 
-... as well as conda:
+or with conda:
 
 ```bash
 conda install -c conda-forge pymmcore-plus
 ```
 
 ## Installing Micro-Manager Device Adapters
 
-Just like underlying the [`pymmcore`](https://github.com/micro-manager/pymmcore)
-that this library, `pymmcore-plus` relies on the device adapters and C++ core
-provided by
+Just like the underlying [`pymmcore`](https://github.com/micro-manager/pymmcore) library,
+`pymmcore-plus` also relies on the device adapters and C++ core provided by
 [mmCoreAndDevices](https://github.com/micro-manager/mmCoreAndDevices#mmcoreanddevices).
-There are two ways to do this:
+They can be installed in two ways:
 
 1. **Use the `mmcore` command line tool**
 
-    If you've installed with `pip install "pymmcore-plus[cli]"`,
-    this library provides a quick way to install the latest version of
-    micro-manager:
+    If you've installed with `pip install "pymmcore-plus[cli]"`, this library provides
+    a quick way to install the latest version of micro-manager:
 
     ```bash
     mmcore install
     ```
 
     This will download the latest release of micro-manager and place it in the
-    pymmcore-plus folder. If you would like to modify the location of the
+    `pymmcore-plus` folder. If you would like to modify the location of the
```

Comment:
this actually isn't correct anymore... so while we're here, might as well update it to be correct.  explain that it goes into the user data folder (I think there's more info in the `install()` docstring)

Reply:
I've updated `install.md` with the new destination and I wanted to add an example path. I've used this
`C:\Users\UserName\AppData\Local\pymmcore-plus\mm` but I'm not sure if it is correct, I need to check in a windows pc.